locales/ru_RU/pcsx2_Main.po

```diff
@@ -1680,12 +1691,10 @@ msgid "Boot CDVD (&fast)"
 msgstr "Запустить CD/DVD (быстро)"
 
 #: pcsx2/gui/MainFrame.cpp:687
-#, fuzzy
 msgid "Boo&t ISO (full)"
 msgstr "Запустить CD/DVD (все этапы)"
 
 #: pcsx2/gui/MainFrame.cpp:688
-#, fuzzy
 msgid "Boot the VM using the current ISO source media"
 msgstr ""
 "Запускает виртуальную машину PS2, используя выбранный DVD-диск или ISO-файл."
```

Comment:
These three (Boot ISO (full), Boot ISO(fast), and the VM tooltip from above don't appear translated either.